modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1051,6 +1051,76 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Label, Output and Formula Builder can be edited after the Custom Object be published"
+	@priority = 4
+	test CanEditFormulaBuilderOnCustomObject {
+		task ("Given: A Custom Obejct with Formula Field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176842",
+				objectName = "CustomObject176842",
+				pluralLabelName = "Custom Objects 176842");
+		}
+
+		task ("When: A formula field on custom objects is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176842");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula Field",
+				fieldType = "Formula",
+				formulaFieldOption = "Decimal");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject176842");
+		}
+
+		task ("And when: the Formula Field can be edited only Label, Output and Formula Builder") {
```

Comment:
Use only `When` here:

`When: the Formula Field can be edited only Label, Output and Formula Builder") {`